src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -15,6 +15,10 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
+  <PropertyGroup Label="MultilingualAppToolkit">
+    <MultilingualAppToolkitVersion>4.0</MultilingualAppToolkitVersion>
+    <MultilingualFallbackLanguage>en-US</MultilingualFallbackLanguage>
+  </PropertyGroup>
```

Comment:
That's a good idea.  @weshaggard would you suggest making a new .targets file in buildtools\src\Microsoft.DotNet.Build.Tasks\Targets for localization?  I'm thinking it would include the two properties at this point in the code, and then also the Import and MATPrerequisite target at lines 248-251.